coralproject/talk
# [CORL-3186]: Update docs packages and add lunrjs search

## What does this PR do?

These are changes that are looking to update Coral's docs to the latest version of Docusaurus, plus switches out algolia search in favor of using `lunrjs` for search functionality. It also updates how we are generating the API docs, now using `@graphql-markdown/docusaurus`.

Everything should be basically the same. The API docs have a slightly different layout, with `Operations` and `Types` categories now.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

You can run `pnpm run build` and `pnpm run serve` in `docs` to see the docs locally.

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
